scripts/init-remote.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
```

Comment:
@otakup0pe Could you put some boilerplate at the top for the expected format that this is called? Looks like it is looking for 2 args, and it is not clear what they are intended to be, or that they are position dependent.

Reply:
Added some inline help and updated the `README`.